### ISSUE TITLE
How to add column from virtual entity?

### ISSUE THREAD

**Issue by Vidmio** [KorzhCom/EasyData#63]
In samples, in Products list one column is CategoryId. I would like to add  CategoryName in this view of Products. Is that possible? How to do it?

**Comment by antifree** [comment]
Hello, 

Unfortunately, it is not possible now. CategoryId is a primary key to the Categories table. It is not the property of a related Product entity. 

We will investigate if it is possible to add such enhancment

**Comment by howest-alexander** [comment]
I would realy like this feature too, we need this feature to make the project usable for us.

thanks a lot.

**Comment by korzh** [comment]
We are working on the new version of EasyData that would allow defining any SQL statement as a data source. I believe this feature will allow us to implement the functionality you requested as well.

**Comment by howest-alexander** [comment]
Yes, thanks a lot, then you also need the possibility to set the name of the field in that query as the navigation id.

Van: Sergiy Korzh ***@***.***>
Verzonden: zondag 22 januari 2023 4:12
Aan: KorzhCom/EasyData ***@***.***>
CC: Bourgois Alexander ***@***.***>; Manual ***@***.***>
Onderwerp: Re: [KorzhCom/EasyData] How to add column from virtual entity? (#63)


We are working on the new version of EasyData that would allow defining any SQL statement as a data source. I believe this feature will allow us to implement the functionality you requested as well.

-
